docs/contributing/Android-API-And-Related-Versions.md

```diff
@@ -0,0 +1,15 @@
+# Android Api and Related versions
+
+## Overview
+
+The most important thing is for any test, project, or app to have compatible versions. 
+If versions need to be different then a comment explaining why and what is being evaluated should be included. 
```

Comment:
Is there an example of this we can link to, or even quote:

> ```gradle
> // Normally we use 3000, but 4000 verifies the SuperAndroid feature.
> // See XYZ
> compileSdk: 4000 
> ```

Reply:
That can be included later. I am literally making up this requirement now and to my knowledge no projects currently do this.

Reply:
Ack.